### DISCUSSION TITLE
https://zhangzhao219.github.io/2022/12/11/advanced-ai-final/

### DISCUSSION THREAD

**Discussion by giscus** [zhangzhao219/zhangzhao219.github.io#3] [Announcements]
# https://zhangzhao219.github.io/2022/12/11/advanced-ai-final/

《高级人工智能》期末复习

https://zhangzhao219.github.io/2022/12/11/advanced-ai-final/

**Comment by Ming-er** [comment]
17-18 年 第 14 题：
A. 为什么不对？如果摇臂次数固定的话，优化每次的期望收益等价于优化累积奖励最大；
C、D 收敛怎么理解？

**Reply by Ming-er** [reply]
KB entails $\alpha$ if and only if KB $\Rightarrow \alpha$ is valid
这个定理是在描述语义蕴含与语法推演的关系吗？

**Reply by zhangzhao219** [reply]
A感觉是对的
错的应该是D，$\epsilon$永远有探索的步骤，不会像贪心策略一样只利用
UCB本质是对Qt(a)进行估计，尝试的越多就越准，越向贪心收敛

**Comment by Ming-er** [comment]
definite 误写为 defeinite

**Reply by zhangzhao219** [reply]
已经修改
